src/main/java/storagesystem/viewcontroller/inventory/DetailedItemViewController.java

```diff
@@ -0,0 +1,359 @@
+package storagesystem.viewcontroller.inventory;
+
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.Event;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.*;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Pane;
+import javafx.stage.FileChooser;
+import storagesystem.model.*;
+import storagesystem.services.PictureHandler;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Controls a detailed view of an item. Can be used to book an item.
+ *
+ * @author Jonathan Eksberg, Carl Lindh
+ */
+
+//todo refactor and make code good looking.
+public class DetailedItemViewController extends AnchorPane {
```

Comment:
Name not consistent with other parts of application. Should be ItemDetailViewController. Possible also specifying Inventory in some way as to not confuse it with controllers and views in /itemview vs /inventory.

Reply:
Bytte namn till ItemDetailViewController